uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/ColumnRenderingStrategyMerged.java

```diff
@@ -241,40 +243,34 @@
         return commands;
     }
 
-    private boolean isCollapsedRowMultiValue(final GridData model,
-                                             final GridColumn<?> column,
-                                             final GridCell<?> cell,
-                                             final int rowIndex) {
+    protected static boolean isCollapsedRowMultiValue(final GridData model,
+                                                      final GridColumn<?> column,
+                                                      final GridCell<?> cell,
+                                                      final int rowIndex) {
         GridRow row;
         int rowOffset = 1;
         final int columnIndex = column.getIndex();
 
         //Iterate collapsed rows checking if the values differ
         while ((row = model.getRow(rowIndex - rowOffset)).isCollapsed()) {
             final GridCell<?> nc = row.getCells().get(columnIndex);
-            if (nc == null) {
```

Comment:
This introduces a slight change in operation... if ```cell == null``` and ```nc == null``` this **used to return** ```true``` whereas now it could potentially return ```false``` (if ```cell == null``` and throughout the checks all other cells are ```null```)... I will run locally and report back.

Reply:
@jomarko I can't get it to render anything _wrong_ so all seems OK.